diff -uNr mount-gtk-1.0.6-old/src/dialogs.cpp /home/unity/Scaricati/mount-gtk-1.0.6/src/dialogs.cpp
--- mount-gtk-1.0.6-old/src/dialogs.cpp	2010-02-12 12:21:28.000000000 +0100
+++ mount-gtk-1.0.6/src/dialogs.cpp	2010-03-07 15:57:31.000000000 +0100
@@ -28,7 +28,7 @@
                        WinBase(caption, prog_config.window_icon_h,
 			       modal, parent_p,
 			       GTK_WINDOW(gtk_message_dialog_new(0, GtkDialogFlags(0),
-					  message_type, GTK_BUTTONS_CLOSE, text))) {
+					  message_type, GTK_BUTTONS_CLOSE, "%s", text))) {
 
   // make further specialisations for a message dialog object
 
